### PR TITLE
Issue/use admin sortable

### PR DESCRIPTION
**Pull request by mkoistinen**
This PR is based against #71, so commit that first.

### CODE REVIEW

File: aldryn_faq/models.py [hunk 1]
```diff
@@ -189,15 +190,19 @@ class Meta:
...
 
+    def __str__(self):
+        question_count = self.questions.count()
+        return '{0} {1} selected'.format(
+            question_count,
+            "question" if question_count == 1 else "questions"
+        )
+
 
 class CategoryListPlugin(CMSPlugin):
```
Comment (czpython) [inline aldryn_faq/models.py]: make sure to add ```default=0``` to the ```position``` field, as required by adminsortable2

**Comment by czpython** [comment]
Good to merge once comment is addressed.

**Comment by mkoistinen** [comment]
Also, rebased against change made on #71.

**Comment by czpython** [comment]
:+1:

**Comment by mkoistinen** [comment]
Rebased now against master.

**Pull request merged**